docs/source/getting_started.rst

```diff
@@ -13,21 +13,46 @@ If you're unfamiliar with the concepts of static and dynamic type checking,
 be sure to read this chapter carefully, as the rest of the documentation
 may not make much sense otherwise.
 
-Installing and running mypy
-***************************
+Installing mypy
+***************
 
-Mypy requires Python 3.9 or later to run.  You can install mypy using pip:
+Mypy requires Python 3.9 or later to run, and can be installed through either pip or uv.
```

Comment:
This sentence also implicitly says "we do support `pip` and `uv`, but may not support other package managers", which is a false impression - `mypy` does not care which installation method was used, `apt-get install mypy` works on Ubuntu as well (even though it installs an ancient version). Perhaps

```suggestion
Mypy requires Python 3.9 or later to run, and can be installed via pip, uv or another package manager.
```

Here's how ruff explains its installation (note that they have to mention uv as that's another tool from Astral.sh, good for marketing, "recommended" is there for the same reason): https://docs.astral.sh/ruff/installation/